.github/workflows/bindings_java.yml

```diff
@@ -44,6 +44,19 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      # spotless requires JDK 11 or later
```

Comment:
We can split a seperate `check` job out of `test`

Reply:
Of course :D

Reply:
Updated.